hardware/Schematic Design Guide.md

```diff
@@ -0,0 +1,148 @@
+
+  For more information, please refer to application notes: [AN0948](https://www.silabs.com/documents/public/application-notes/an0948-power-configurations-and-dcdc.pdf)
+- Debug connector and reset pin:
+  - Serial wire debug
+
+    The Serial Wire Debug (SWD) interface is supported by all EFR32 Wireless Gecko Series devices and consists of the SWCLK (clock input) and SWDIO (data in/out) lines, in addition to the optional SWO (serial wire output). The SWO line is used for instrumentation trace and program counter sampling, and is not needed for flash programming and normal debugging. However, it can be valuable in advanced debugging scenarios, and designers are strongly encouraged to include this along with the other SWD signals.And more there are VCOM port and PTI trace port in the mini simplicity studio debug port. This allow the debug port to do powerful program, control, and trace functions.
+  The debug port map and description is as following:
+  ![](../files/HW-Schematic-Design-Guide/SWD-Debug-Port.png)
+  Most commonly it recommends to reserve this debug port in custom boards to facilitate the debug and test purpose.
+
+  - JTAG debug
+
+    EFR32 Wireless Gecko Series devices optionally support JTAG debug using the TCLK (clock), TDI (data input), TDO (data output), and TMS (input mode select) lines. TCLK is the JTAG interface clock. TDI carries input data, and is sampled on the rising edge of TCLK. TDO carries output data and is shifted out on the falling edge of TCLK. Finally, TMS is the input mode select signal, and is used to navigate through the Test Access Port (TAP) state machine.
+    The 10-pin Cortex debug port is defined as following:
+    ![](../files/HW-Schematic-Design-Guide/Cortex-Debug-Port.png)
+
+  For more information for debug connecors, please refer to [AN0958](https://www.silabs.com/documents/public/application-notes/an958-mcu-stk-wstk-guide.pdf)
+  - Reset pin
+
+    There are a weak pull-up resistor and a low pass filter at reset pin inside the chip. It will allow reset pin floated and prevent noise glitches from outside interfer. The RESETn must not be connected through an external pull-up to an active supply, otherwise driven high as this could damage the device.
+
+- External clock source:
+
+  EFR32 Wireless Gecko Series devices support different external clock sources to generate the high and low frequency clocks in addition to the internal LF and HF RC oscillators. The possible external clock sources for both the LF and HF domains are external oscillators (square or sine wave) or crystals/ceramic resonators.
+
+  - Low Frequency Clock Sources
+
+    An external low frequency clock can be supplied from a crystal/ceramic resonator or from an external clock source. It can source a low-frequency clock from an external source such as a TCXO or VCXO.
```

Comment:
Shall we delete the last "from" ?
An external low frequency clock can be supplied from a crystal/ceramic resonator or an external clock source.